Library/Formula/libmxml.rb

```diff
@@ -20,6 +20,10 @@ def install
                           "--enable-shared",
                           "--prefix=#{prefix}"
     system "make"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    assert shell_output("#{bin}/mxmldoc --version").include?("2.9")
```

Comment:
Agreed and updated to check exit code only.

A more complicated test is of course possible by writing a C program (or by somehow building and running the included `testmxml.c`), but it's beyond my current capacity, sorry.